spec/Section 4 -- Introspection.md

```diff
@@ -352,11 +352,11 @@ Fields
 
 #### List
 
-Lists represent sequences of values in GraphQL. A List type is a type modifier:
+Lists represent sequences of values in GraphQL. A List type is a wrapped type:
```

Comment:
I think this phrasing is ambiguous/confusing. A list type is a "wrapping type" but an instance of a list type is a "wrapped type". Within introspection we only see instances, so this usage is probably right, but this phrasing relative to "A List type" makes this distinction a bit less clear IMO.

What do you think about:

> Lists represent sequences of values in GraphQL. A List wrapped type expects a List value. It wraps another type in the `ofType` field (the "inner type"), which defines the type of each item in the list.

This replaces "A List type is a wrapped type" with "A List wrapped type" - using your definition

Reply:
Scratch this feedback. I'm working on a broader change based on yours that applies this new terminology broadly and I think I can resolve this at the definition of "List type"

Reply:
> This replaces "A List type is a wrapped type" with "A List wrapped type"

This reads a lot better :+1: 

> Scratch this feedback. I'm working on a broader change based on yours that applies this new terminology broadly and I think I can resolve this at the definition of "List type"

Okay :+1: